databases/business-sync/fragment-shoebox-snippet.js

```diff
@@ -0,0 +1,40 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc),
+  typeFilter: function(doc, oldDoc) {
+    return createBusinessEntityRegex('shoeboxSnippet\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+').test(doc._id);
```

Comment:
My suggestions are to go with either shoeboxItem or sourceData. I prefer sourceData on a 6/10 strength :D I'd rather keep the metaphor out of the key names because it could become dated quickly and might need to be explained to every new dev. On the other hand, if we're always talking shoebox, then shoeboxItem/Snippet might come to mind first.

Does @cclark have any thoughts?